fog-aws.gemspec

```diff
@@ -28,4 +28,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'fog-json',  '~> 1.0'
   spec.add_dependency 'fog-xml',   '~> 0.1'
   spec.add_dependency 'ipaddress', '~> 0.8'
+  spec.add_dependency 'mime-types'
```

Comment:
My preference is to keep this out of the fog/fog-aws gemspec (since different versions are needed depending on which ruby you are using). We can have a guarded/warning if you try to do something that needs it without having it loaded yet.